app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
All custom exceptions should be inherited from Exception class not BaseException. 
Please, refer [here](https://docs.python.org/3/library/exceptions.html#BaseException) for more info.